app/scripts/migrations/126.test.ts

```diff
@@ -0,0 +1,50 @@
+import { migrate, version } from './126';
+
+const oldVersion = version - 1;
+
+describe(`migration #${version}`, () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('Does nothing if `providerConfig` is not in the network controller state', async () => {
+    const oldState = {
+      NetworkController: {
+        selectedNetworkClientId: 'mainnet',
+      },
+    };
+
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: oldState,
+    });
+
+    expect(transformedState.data).toStrictEqual(oldState);
+  });
+
+  it('Removes providerConfig from the network controller state', async () => {
```

Comment:
To ensure that the new logic correctly handles the presence and absence of the providerConfig property, please add unit tests that cover both scenarios: one where providerConfig exists and is successfully deleted, and another where it doesn't exist and return the existing state ( see the comment after )

Reply:
There's already tests for both cases.  providerConfig being in state and not in state

`Does nothing if providerConfig is not in the network controller state`